### PR TITLE
fozzie-components@v7.39.0 - Molecules - Remove duplicate dependencies #globalconfig

### PR DESCRIPTION
**Pull request by siggerzz**
This PR removes the following dev dependencies from molecule components, as there are versions already defined in the root of the project.
```
    "@vue/cli-plugin-babel"
    "@vue/cli-plugin-unit-jest"
    "@vue/test-utils"
```

### CODE REVIEW

File: packages/components/molecules/f-media-element/package.json [hunk 1]
```diff
@@ -54,8 +54,6 @@
   },
   "devDependencies": {
     "@vue/cli-plugin-babel": "4.5.16",
```
Comment (xander-marjoram) [inline packages/components/molecules/f-media-element/package.json:56]: Is this one still needed?

File: packages/components/molecules/f-mega-modal/package.json [hunk 2]
```diff
@@ -57,8 +57,6 @@
     "@justeat/f-wdio-utils": "1.x",
     "@justeattakeaway/pie-icons-vue": "1.0.0",
     "@vue/cli-plugin-babel": "4.5.16",
```
Comment (xander-marjoram) [inline packages/components/molecules/f-mega-modal/package.json:59]: Same here

File: packages/components/molecules/f-media-element/package.json (hunk 1 above)
Comment (siggerzz) [reply packages/components/molecules/f-media-element/package.json:56]: Yep, was getting issues with the babel optional chaining without it. Will be looking at a fix for this in a future ticket 👍🏼

File: packages/components/molecules/f-mega-modal/package.json (hunk 2 above)
Comment (siggerzz) [reply packages/components/molecules/f-mega-modal/package.json:59]: ditto

**Comment by jamieomaguire** [comment]
NIT: The pr description talks about atoms instead of molecules :P

**Review by jamieomaguire — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**